libs/environment/src/components/arbitrum-link.tsx

```diff
@@ -0,0 +1,48 @@
+import { ExternalLink } from '@vegaprotocol/ui-toolkit';
```

Comment:
This component is not required. EtherscanLink has already been modified to handle other chains. (We should rename it later though)